### PR TITLE
javadoc for the utility functions

### LINKED ISSUE #3: Add javadocs to describe the purpose of the two utility methods
Pinger uses two utility methods to cut down on code repetition and to make it easier to understand. 

The following methods do not have any documentation that explains what they do. 
https://github.com/acmpesuecc/pinger/blob/8b164118db9035a5bd6fe8775164dbcfefa546e0/src/main/java/pinger/Pinger.java#L32 https://github.com/acmpesuecc/pinger/blob/8b164118db9035a5bd6fe8775164dbcfefa546e0/src/main/java/pinger/Pinger.java#L46

This can be solved be adding javadoc comments above the method signature as so
```
/**
 * write documentation here
 */
```

### PR DESCRIPTION
**Pull request by sreechand2002**

### CODE REVIEW

**Review by BoogieMonster1O1 — CHANGES_REQUESTED** [review]
The documentation should be in a comment above the method
e.g.:
```java
/**
 * docs go here
 */
public void foo() {
}
```

**Comment by sreechand2002** [comment]
checkout now

File: src/main/java/pinger/Pinger.java [hunk 1]
```diff
@@ -45,7 +45,9 @@ public static void main(String[] args)  {
             } 
         }while(ch == 'y');
     }
-
+    /*The readalllines() is used to take input repetatively
```
Comment (BoogieMonster1O1) [inline src/main/java/pinger/Pinger.java]: javadoc comments should have two asterisks and should not have any text on the first line. 
```java
/**
 * docs go here
 */
```
also the `readAllLines` function reads all lines from any input stream. it isn't used to read input from the user.

File: src/main/java/pinger/Pinger.java [hunk 2]
```diff
@@ -60,7 +62,10 @@ private static String readAllLines(InputStream stream) throws IOException {
             return builder.toString();
         }
     }
-
+    /*
+    The addfield() is used to assign the property values which
+    were present in the user input
```
Comment (BoogieMonster1O1) [inline src/main/java/pinger/Pinger.java]: user input -> returned input

**Review by BoogieMonster1O1 — CHANGES_REQUESTED** [review]

**Comment by sreechand2002** [comment]
changed it

**Comment by BoogieMonster1O1** [comment]
Please fix the javadoc comments 
They should begin with two asterisks

File: src/main/java/pinger/Pinger.java [hunk 3]
```diff
@@ -45,7 +45,9 @@ public static void main(String[] args)  {
             } 
         }while(ch == 'y');
     }
-
+    /**The readalllines() is used to take input repetatively
```
Comment (BoogieMonster1O1) [inline src/main/java/pinger/Pinger.java]:
```suggestion
    /**
    The readalllines() is used to take input repetatively
```

**Comment by sreechand2002** [comment]
is it fine now?

File: src/main/java/pinger/Pinger.java (hunk 3 above)
Comment (BoogieMonster1O1) [inline src/main/java/pinger/Pinger.java]:
```suggestion
    /**The readAllLines() is used to read input from a stream until there is no more data left
```

File: src/main/java/pinger/Pinger.java [hunk 4]
```diff
@@ -60,7 +62,10 @@ private static String readAllLines(InputStream stream) throws IOException {
             return builder.toString();
         }
     }
-
+    /**
+    The addfield() is used to assign the property values which
```
Comment (BoogieMonster1O1) [inline src/main/java/pinger/Pinger.java]:
```suggestion
    The addField() is used to assign the property values which
```
fix capitalization

**Review by BoogieMonster1O1 — CHANGES_REQUESTED** [review]

**Comment by sreechand2002** [comment]
can you review it now

**Review by BoogieMonster1O1 — APPROVED** [review]

**Comment by Rishabkashyap14** [comment]
!bounty 30

**Pull request merged**